<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckEditor</class>
 <widget class="QWidget" name="DeckEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="cardsTable">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addCardButton">
      <property name="text">
       <string>Добавить карточку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="englishWordInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="translationInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="removeCardButton">
      <property name="text">
       <string>Удалить карточку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
